### DISCUSSION TITLE
Allowing world name in backup file

### DISCUSSION THREAD

**Discussion by devsi** [mbround18/valheim-docker#233] [Ideas]
I was switching between worlds from `saves/worlds/` on my server and noticed that the backups don't reflect the world change if the world alters. This makes it impossible to tell which backup file belongs to which world.

Could the backup file use also contain the world name to avoid this problem?

Alternatively, could the filename format be specified as an ENV VAR in docker-compose so we have custom control over the name?

**Comment by mbround18** [comment]
Backups are per your whole save folder and not per world. Thus, if you are switching world names its best to check the `tar.gz` file for the world you wish to restore. This is because many servers utilize multiple worlds. However, adding a tag for an environment variable to alter what the backups called for usability isn't a bad idea. Though, going forward please open a Discussion and not an issue. 

Discussions help with readability and discoverability within the repo <3 Transferring to discussions...

**Reply by devsi** [reply]
Thanks! I didn't realize it was everything in `saves/` although that makes a lot of sense. Why would you back up half the data and not all of it. If I was reading the `sh` file closer I may have noticed that.
